rspec-rails.gemspec

```diff
@@ -21,7 +22,7 @@ Gem::Specification.new do |s|
     'source_code_uri'   => 'https://github.com/rspec/rspec-rails',
   }
 
-  s.files            = `git ls-files -- lib/*`.split("\n")
+  s.files            = Rake::FileList["lib/**/{.*,*}"].exclude(*File.read(".gitignore").split).reject { |f| File.directory?(f) }
```

Comment:
Is there a difference between `Rake::FileList.ne` and `Dir[]` for our use case?

Reply:
I don't think so. I prefer `Dir` over `Rake::FileList`. But to respect the `.gitignore`, I went ahead with `Rake::FileList`, do you prefer `Dir`? I'll be happy to implement whatever you're happy with! 😄

Reply:
I got it. Surely using `Dir` would be nice. But if it happens that we need to `exclude`, `FileList` is a better option.
Let's see if in the other repos `exclude` makes any difference, let's go with `FileList`.

Reply:
You want me to open PRs for all the repositories now itself?
I'd say, let's go one by one instead? 🙈

Reply:
The only reason I'm asking is to be able to make this code similar across apps.
If it happens that some other repo need to `exclude`, it would be nice to use `FileList` everywhere, not only in that other repo.

Reply:
Okay, so I'll open PRs this weekend for:
- rspec-support 
- rspec-dev
- rspec-expectations
- rspec-mocks
- rspec-its

Please let me know if I'm missing something 😅

Reply:
`rspec-dev` isn't backing any gem, but https://github.com/rspec/rspec does.

Optional repos you may want to include:
 - https://github.com/rspec/rspec-collection_matchers
 - https://github.com/rspec/rspec-its
 - https://github.com/rspec/rspec-activemodel-mocks
 - https://github.com/rspec/rspec-legacy_formatters
 - https://github.com/rspec/rspec-autotest